automation/fasttest.sh

```diff
@@ -61,7 +61,9 @@ if [ -z "$db_id" ] || [ -z "$redis_id" ] || [ -z "$api_id" ]; then
 	redis_id=$(runredis '-p 6378:6379')
 	echo $redis_id >>"$CONFIG_FILE"
 
-	api_id=$(runapi $IMAGE_NAME $db_id $redis_id " -p 9228:9229 -v $(pwd):/usr/src/app")
+	# Inject the SUPERUSER_EMAIL and SUPERUSER_PASSWORD env vars so that test
+	# suites have a set of user credentials to work with
+	api_id=$(runapi $IMAGE_NAME $db_id $redis_id " -e SUPERUSER_EMAIL=test@example.com -e SUPERUSER_PASSWORD=foobarbaz -p 9228:9229 -v $(pwd):/usr/src/app")
```

Comment:
These options should be in `runapi` in common.sh instead

Reply:
@Page-  Is `runapi` ever used for non-testing purposes?

Reply:
Nope, it's specifically for testing; `common.sh` is for things shared between ci testing in `test.sh` and local `fasttest.sh` (which just skips some steps like linting that aren't useful in the context of locally iterating on a test)